compose/common.yml

```
@@ -176,7 +176,8 @@ services:
         '--node-config',
         '/config/cardano-node/config.json',
         '--log-level-websocket',
-        'Warning'
+        'Warning',
+        '--include-cbor'
```

We don't need the granular ones
```suggestion
        '--include-transaction-cbor'
```